ChIP-Seq/tutorials/TAL1_binding_site_identification.md

```diff
@@ -0,0 +1,420 @@
+Identification of the binding sites of the T-cell acute lymphocytic leukemia protein 1 (TAL1) with ChIP-sequencing
+=============
+
+:grey_question: ***Questions***
+
+- *How is raw chIP-seq data processed and analyzed?*
+- *What are the binding sites of Tal1?*
+- *Which genes are regulated by Tal1?*
+
+:dart: ***Objectives***
+
+- *Inspect read quality with FastQC*
+- *Perform read trimming with Trimmomatic*
+- *Align trimmed reads with BWA*
+- *Assess quality and reproducibility of experiments*
+- *Identify Tal1 binding sites with MACS2*
+- *Determine unique/common Tal1 binding sites from G1E and Megakaryocytes*
+- *Identify unique/common Tal1 peaks occupying gene promoters*
+- *Visually inspect Tal1 peaks with Trackster*
+
+:heavy_check_mark: ***Requirements***
+
+- *Galaxy introduction* (add link)
+- *NGS-QC* (add [link](../../NGS-QC/slides/dive_into_qc.md))
+- *NGS-mapping* (add [link](../../NGS-mapping/slides/dive_into_mapping.md))
+- *Trackster* (add link)
+
+:hourglass: ***Time estimation*** *3h*
+
+# Content
+
+* [Introduction](#introduction)
+* [Hands-on example of ChIP-seq analysis](#example)
+  - [Step 1: Quality control](#step1)
+  - [Step 2: Trimming/clipping reads](#step2)
+  - [Step 3: Aligning reads to a genome](#step3)
+  - [Step 4: Assessing correlation between samples](#step4)
+  - [Step 5: Assessing GC bias](#step5)
+  - [Step 6: Assessing IP strength](#step6)
+  - [Step 7: Determining Tal1 binding sites](#step7)
+  - [Step 8: Inspection of Tal1 peaks](#step8)
+  - [Step 9: Identifying unique/common Tal1 peaks](#step9)
+  - [Additional optional analyses](#stepA)
+* [Concluding remarks](#conclusion)
+* [Useful literature](#literature)
+  
+<a name="introduction"/></a>
```

Comment:
These are links marking the various sections of the tutorial that can be jumped to by clicking on the link in the contents table at the top of the tutorial. If there is a more MD-style way to do this, I'll change it!